app/experimenter/nimbus-ui/src/types/globalTypes.ts

```diff
@@ -7,13 +7,10 @@
 // START Enums and Input Objects
```

Comment:
Why are these constants defined here?  These should only be fetched from the graphql API?

Reply:
These are a part of the auto-generated types, they're enums pulled from the GQL API schema by the code generator

Reply:
https://jira.mozilla.com/browse/EXP-536

Reply:
Thanks @jodyheavener !